css/properties/background-clip.json

```diff
@@ -168,13 +168,18 @@
               "chrome": {
                 "version_added": "3",
                 "partial_implementation": true,
-                "notes": "The <code>text</code> value is only supported by <code>-webkit-background-clip</code> (and not by <code>background-clip</code>)."
+                "notes": "The <code>text</code> value is only supported by <code>-webkit-background-clip</code> (and not by <code>background-clip</code>; see <a href='https://crbug.com/1339290'>bug 1339290</a>)."
               },
               "chrome_android": "mirror",
               "edge": [
                 {
                   "version_added": "15"
                 },
```

Comment:
I specifically have this entry because from version 15 until 18, Edge had full support for the value being supported with the unprefixed version, but ever since version 79, that's the same thing as for Chrome.

Reply:
Okay, then maybe this entry should come second and the 79 entry first. The first entry is the one that MDN renders in the table cell and additional entries are only shown when you expand the table cell.

Reply:
Okay. This would revert 7b58a03b08d7c723f15d17be8f54d0d2d5d8277b, which was by running `npm run fix` on the file. However, I think this should be a specific case that `npm run fix` didn't actually "fix".

Reply:
Applied with commit 412bffecec5870a794fd3c7fb80ead6e2c7b5c10.